api/v1alpha2/virtualmachine_network_types.go

```diff
@@ -32,7 +32,7 @@ type VirtualMachineNetworkInterfaceSpec struct {
 	// ensure the provided device name does not conflict with any other devices
 	// inside the guest, ex. dvd, cdrom, sda, etc.
 	//
-	// +kubebuilder:validation:Pattern=^\w\w+$
+	// +kubebuilder:validation:Pattern=^[a-z0-9][a-z0-9]+$
```

Comment:
I don't know why I used two `\w`. Can you please see if `^[a-z0-9]{1,}$` works as well?

Reply:
Spoke to @akutz offline. We want atleast two characters so `^[a-z0-9][a-z0-9]+$` or `^[a-z0-9]{2,}$` should do. I will test this out.